float64[] viapoints
int16 num_viapoints
float32 max_time
float32 max_x
float32 interval_time
string[] obstacles
int16 num_obstacles
float32[] obstacle_positions
---
float32 reward
float32 distance_covered
float32 obstacle_displacement
